### ISSUE TITLE
Support to set item slots across all the item sets?

### ISSUE THREAD

**Issue by xoxfaby** [dustinblackman/Championify#331]
New patch added item slots preference to items, would be great to be able to apply this to all item sets added by championify. 

http://na.leagueoflegends.com/en/news/game-updates/patch/patch-717-notes

**Comment by SalGnt** [comment]
+1

**Comment by dustinblackman** [comment]
Ah this is nice! However that may only be for item sets created in the client rather then imported. I'll check either way. Thanks!

**Comment by xoxfaby** [comment]
"preferredItemSlots":[{"id":"2031","preferredItemSlot":1},{"id":"3142","preferredItemSlot":2},{"id":"3814","preferredItemSlot":0}]

honestly, looking at how simple that is, I'm surprised you haven't implemented it yet. 
If nothing else give a way to append some text to every file so I can put my preferred item slots in there.

**Comment by dustinblackman** [comment]
@xoxfaby I haven't tested `preferredItemSlots`, nor is it in the docs. I'm guessing it was found in item sets developed by the launcher?

**Comment by xoxfaby** [comment]
Yup, took about 5 minutes to find by making one in the launcher and seeing it.

**Comment by dustinblackman** [comment]
While I like the idea of it, I won't have the time anytime soon to add a UI in order for users to customize item slots. PRs are welcome.

**Comment by xoxfaby** [comment]
I would try but I haven't worked in Java in a long time and I have no idea what to even work in for this.

**Comment by WereGoingOcean** [comment]
@xoxfaby if you manually add the `preferredItemSlots` section to a Championify item set does it work for you in game? When I tried it the Championify generated set wouldn't put the items in their preferred slot, only the client created ones worked.

**Comment by volt255** [comment]
@WereGoingOcean: you are correct. I tried manually adding the section preferredItemSlots with a simple "replace in files" from notepad++ . I simply replace all files with filename: CIFY*.json and I replaced the text:

"mode": "any",

with:

"mode": "any",
	"preferredItemSlots": [
		...
	],
from an item set with all of my preferred Item Slots. BUT, when i go to the practice tool, only the item sets created by the client work, Apparently its a game issue when working with third party item sets it does not load the "preferredItemSlots" section.
